nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -1038,19 +1288,15 @@ def __init__(
                 megatron_legacy=megatron_legacy,
                 bias=bias,
                 headscale=headscale,
+                activations_checkpoint_granularity=activations_checkpoint_granularity,
+                sequence_parallel=sequence_parallel,
             )
-            # Normformer normalization
```

Comment:
Why remove the `self.post_attention_normformer_norm` ? In the forward method at L1245-L1246, it is used there.

Reply:
Not intended.